test/unit/gateways/mundipagg_test.rb

```diff
@@ -38,6 +38,8 @@ def setup
       billing_address: address,
       description: 'Store Purchase'
     }
+
+    @gateway_response_error = 'Esta loja n??o possui um meio de pagamento configurado para a bandeira VR'
```

Comment:
I haven't been able to reproduce one of these errors, as they likely require a configuration discrepancy. There is thus no corresponding remote test. However, the code and test were written based on examination of a real such error encountered by a user.